microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -300,35 +275,22 @@ def run_testpmd_concurrent(
     task_manager = start_testpmd_concurrent(node_cmd_pairs, seconds, log, output)
     if rescind_sriov:
         time.sleep(10)  # run testpmd for a bit before disabling sriov
-        test_kits = node_cmd_pairs.keys()
 
-        # disable sriov
-        for node_resources in test_kits:
-            node_resources.nic_controller.switch_sriov(enable=False, wait=False)
+        test_kits = node_cmd_pairs.keys()
 
-        # wait for disable to hit the vm
+        # disable sriov (and wait for change to apply)
         for node_resources in test_kits:
-            if not node_resources.wait_for_dmesg_output("AN_DISABLE", seconds // 3):
-                fail(
-                    "Accelerated Network disable not found in dmesg"
-                    f" before timeout for node {node_resources.node.name}"
-                )
+            node_resources.nic_controller.switch_sriov(enable=False, wait=True)
 
-        time.sleep(10)  # let testpmd run with sriov disabled
+        # let run for a bit with SRIOV disabled
+        time.sleep(10)
```

Comment:
above switch_sriov already use wait, so it will check the rescind works or not there, no need sleep

Reply:
I do need to collect some data with each state though, so these sleeps are to let the other thread run for a few seconds.